app/assets/javascripts/comment.js

```diff
@@ -31,11 +31,14 @@
 
     if(!$(this).hasClass('bound-keypress')) {
       $(this).addClass('bound-keypress');
-      $(this).find('#text-input').bind('keypress',function(e){
-        if (e.ctrlKey && e.keyCode === 10) {
+
+      $(this).on('keypress', function (e) {
```

Comment:
This looks good! Could you change the event to `keydown`. From the docs for `keypress`: 
```
The keypress event is sent to an element when the browser registers keyboard input. This is similar to the keydown event, except that modifier and non-printing keys such as Shift, Esc, and delete trigger keydown events but not keypress events. Other differences between the two events may arise depending on platform and browser.
```

Since here, CTRL + enter are non-printing keys, we should use the other option.

Reply:
As long as it works, I suggest we keep it that way.